### PR TITLE
Added support for RHEL 7.

### PR DESCRIPTION
**Pull request by bschonec**

### CODE REVIEW

**Comment by bschonec** [comment]
Well, I don't know why my second commit showed up here in the pull request.  GIT, you confuse me at every turn!

File: manifests/init.pp [hunk 1]
```diff
@@ -31,6 +31,11 @@
     owner   => 'root',
     group   => 'root',
     mode    => '0644',
-    content => template('selinux/config.erb'),
+    content => template('selinux/config.erb');
```
Comment (ghoneycutt) [inline manifests/init.pp]: Please revert changes to this file. This is an addition of additional features not related to supporting EL 7. It also broke the build. By grouping your pull requests by topic it makes it easier for the maintainer to accept your code.

File: metadata.json [hunk 2]
```diff
@@ -22,7 +22,8 @@
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
         "5",
-        "6"
+        "6",
+        "7"
```
Comment (ghoneycutt) [inline metadata.json]: 'EL' is Enterprise Linux and applies to the rest of the operatingsystem's listed. Please add the 7 to all of them.

**Comment by ghoneycutt** [comment]
@bschonec Thanks for the work, I made some notes.

BTW: you are submitting from your master branch and should be using a topic branch. This allows you to work on different things - such as EL 7 support and managing /etc/sysconfig/selinux while also staying up to date with upstream (me).

This workflow assumes that you added me as upstream. Which can be done with

`git remote add upstream https://github.com/ghoneycutt/puppet-module-selinux.git`

``` bash
$ git checkout master # get on master branch
$ git pull upstream master # sync your master branch from mine
$ git checkout -b add_support_for_el7 # create a new branch
   ..hack..  # make changes :)
$ git commit -a # commit changes
$ git push origin add_support_for_el7 # push changes to origin (you)
Send a PR via github
```

You can practice here - https://github.com/unixmonkey/learngit

**Comment by bschonec** [comment]
Wow, thanks for that info.  I think that's what I've been missing (by not
doing) for this entire time.

On Thu, Dec 10, 2015 at 9:30 AM, Garrett Honeycutt <notifications@github.com

**Comment by ghoneycutt** [comment]
Thanks @bschonec 

Adding EL7 in PR #34

**Pull request closed without merging**